### PR TITLE
Chore/badge design polish

### PR DESCRIPTION
**Pull request by zencephalon**
**What changed? Why?**
Screenshot 2024-07-26 at 12 48 10 PM
Screenshot 2024-07-26 at 12 48 04 PM

Update badges, make the Talent score show up in the modal.

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx [hunk 1]
```diff
@@ -141,65 +141,76 @@ export const BADGE_INFO: Record<
...
+  score,
+  size,
+  name,
+}: {
+  badge: BadgeNames;
+  claimed?: boolean;
+  score?: number;
+  size: number;
+  name: string;
+}) {
```
Comment (kirkas) [inline apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx:156]: Nit: TS interface

File: apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx [hunk 2]
```diff
@@ -141,65 +141,76 @@ export const BADGE_INFO: Record<
...
+  const showTalentScore = Boolean(claimed && score && badge === 'TALENT_SCORE');
+
+  return (
+    <div className="relative flex h-[160px] w-[160px] items-center justify-center">
+      <Image
```
Comment (kirkas) [inline apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx:161]: Nit: we have ImageWithLoading if you want a nice loading effect

**Review by kirkas — APPROVED** [review]

**Review by kirkas — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for kirkas @ 2024-07-26 17:12:23 UTC
User cannot review their own commit

**Review by brendan-defi — APPROVED** [review]

**Review by brendan-defi — APPROVED** [review]

**Review by brendan-defi — APPROVED** [review]

**Pull request merged**